### PR TITLE
Isdb 349 ccd search api integration

### PR DESCRIPTION
**Pull request by parvezhmcts**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/ISDB-349

### Change description ###

integration with CCD search case api

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/test/java/uk/gov/hmcts/reform/prl/services/cafcass/CafcassCcdDataStoreServiceTest.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+import static uk.gov.hmcts.reform.prl.utils.TestConstants.TEST_AUTHORIZATION;
+
+@RunWith(MockitoJUnitRunner.class)
+public class CafcassCcdDataStoreServiceTest {
+
+    @Mock
+    private CoreCaseDataApi coreCaseDataApi;
+
+    @Mock
+    private AuthTokenGenerator authTokenGenerator;
```
Comment (hmcts-kodatala) [inline src/test/java/uk/gov/hmcts/reform/prl/services/cafcass/CafcassCcdDataStoreServiceTest.java:33]: @Mock
    private AuthTokenGenerator authTokenGenerator; we need remove this , as we are not using

**Review by hmcts-kodatala — COMMENTED** [review]
Pls resolve the review comments

**Review by hmcts-kodatala — APPROVED** [review]
OK

File: src/test/java/uk/gov/hmcts/reform/prl/services/cafcass/CafcassCcdDataStoreServiceTest.java (hunk 1 above)
Comment (parvezhmcts) [reply src/test/java/uk/gov/hmcts/reform/prl/services/cafcass/CafcassCcdDataStoreServiceTest.java]: Done

**Comment by parvezhmcts** [comment]
build is done in jenkins but here it's still showing some checks pending. so closing it and will open new PR.

**Pull request closed without merging**